fhd_core/polarization/stokes_cnv.pro

```diff
@@ -49,6 +49,10 @@ ENDELSE
 n_pix=N_Elements(inds)
 IF Keyword_Set(inverse) THEN p_use=p_map ELSE p_use=p_corr
 
+stokes_mat_term1=[1,1,1,complex(0,1)]
```

Comment:
Please add a comment that this is a subset of the full 4x4 matrices, with the zero terms omitted